netutil.py

```diff
@@ -371,3 +371,13 @@ def disable_ipv6_module(root):
     dv6fd.write("alias net-pf-10 off\n")
     dv6fd.close()
 
+
+from xcp.net.ifrename.dynamic import DynamicRules
```

Comment:
Is it possible/complicated that you do not use DynamicRules here?
The reason why I ask this is because you are now the only user want to use this and we are going to drop this feature from the library.

Reply:
This is the only source of the last order. So it would have to be parsed either by the existing code, or new code. The existing code can be dropped when the upgrade path is not possible.